api/http/internal.go

```diff
@@ -56,11 +56,11 @@ func (h *InternalHandler) ProvisionDevice(c *gin.Context) {
 		Tenant:  tenantID,
 	})
 	err := h.app.ProvisionDevice(ctx, device.ID)
-	switch errors.Cause(err) {
-	case nil:
-		c.Status(http.StatusAccepted)
-	case app.ErrNoConnectionString:
+	switch cause := errors.Cause(err); cause {
+	case nil, app.ErrNoConnectionString:
```

Comment:
how should we treat lack of the connection string?
is that a simple no op, or should we return an error?

Reply:
It's a simple noop. Since the workflow is triggered on all provision_device workflows, in most cases users won't have the connection string configured.